cli/command/container/cp.go

```diff
@@ -98,14 +94,18 @@ func NewCopyCommand(dockerCli command.Cli) *cobra.Command {
 			}
 			opts.source = args[0]
 			opts.destination = args[1]
+			if !cmd.Flag("quiet").Changed {
+				// User did not specify "quiet" flag; suppress output if no terminal is attached
+				opts.quiet = !dockerCli.Out().IsTerminal()
+			}
 			return runCopy(dockerCli, opts)
 		},
 	}
 
 	flags := cmd.Flags()
 	flags.BoolVarP(&opts.followLink, "follow-link", "L", false, "Always follow symbol link in SRC_PATH")
 	flags.BoolVarP(&opts.copyUIDGID, "archive", "a", false, "Archive mode (copy all uid/gid information)")
-	flags.BoolVarP(&opts.quiet, "quiet", "q", !dockerCli.Out().IsTerminal(), "Does not display copying process")
+	flags.BoolVarP(&opts.quiet, "quiet", "q", false, "Suppress progress output during copy. Progress output is automatically suppressed if no terminal is attached")
```

Comment:
Description is a bit lengthy now; I need to think for a shorter description 😅

Reply:
Oh, I think this ok, neither too longo nor too small.